client/my-sites/checkout/src/lib/get-jetpack-product-features.ts

```diff
@@ -201,6 +217,14 @@ export default function getJetpackProductFeatures(
 		];
 	}
 
+	if ( isJetpackGrowthPlan( product.product_slug ) ) {
+		return [
+			...getFeatureStrings( 'growth', translate ),
+			...getFeatureStrings( 'social-basic', translate ),
```

Comment:
I think instead of listing all the Social Basic features here, we could remove this line and just have a line like `Jetpack Social` like you have above with `Jetpack Stats`